### ISSUE TITLE
Should we update the Armory?

### ISSUE THREAD

**Issue by Imagebard** [d3athrow/vgstation13#8373]
There's a small powercreep going on the game, one bit by one bit, so it might as well be time to consider this. 
Aliens aren't weaker to laser weapons anymore. R&D guns are slowly being buffed(Only nerf lately was making the AEG harder to get), and the whole crew can construct weapons thanks to #8010 

That is, _any non-antag part of the crew_ can outgun security given enough time. Engineers can make instakill Railguns, Research has the protolate plus that overpowered as fuck directional explosion gun, and so on.
The question is, what should we put on the armory to make it better?

Some suggestions:
Ricochet Guns for ~~attacking enemies without risking being beaten to death ourselves~~ duck hunting.
Add a copy of all XCOM laser weapons to the rack on the armory, and squaddie armor.

NOTE: Armory contents are different on Metastation, consider checking it.

**Comment by Imagebard** [comment]
![000](https://cloud.githubusercontent.com/assets/7571238/13181626/c2fe9dee-d70c-11e5-87de-f6ef3303f367.png)

Taxi, Deff, Box, and Metastation.

**Comment by viniciuspetch** [comment]
I really think Security's firepower should get a great buff (and really reinforce armory's defenses), and a downgrade on the "illegal" weapons. My biggest concern is Cargo and Research being able to get weapons easier and better than Sec. While Cargo need to do some sketchy maneuvers to open locked crates, Research can make them easily as a result of their obligatory work (researching).

AEGs being transformed into modkits is just the tip of the iceberg, in my opinion all weapons (and maybe all dangerous objects, but I don't think there's any other) made from Research should need a Captain or Sec member to make them operational. Maybe even the same for dangerous crates (adding a extra security measure, like certain crates exploding when being forced, warning when a crate is violated, needing authorization to order those crates, etc).

About the makeshift weapons, I believe the only problem found is some certain weapons, specially the blast cannon. Also, what's the position of space law against non-sec having weapons?

About the armory itself, I always found it so shitty and kind of random. Why only 4 energy guns? Where's the assault rifles and shit? Why the hell there's prisoner's clothes at the locker? The armory need to consider threats like blobs (the reason AEG is needed currently), xenos, spiders, invasions in general. In the end, we/they need to rely on Cargo or Research getting armed instead of Security organizing the militia.

Finally, don't know what to do about xeno weapons. Maybe spawn them broken? Nerf their damage?

TL;DR: buff sec, nerf cargonia and rnd. maybe nerf the new makeshift stuff.
Maybe I said shit, so expect stupid stuff.

**Comment by SonixApache** [comment]
There isn't much in the code to add that wouldn't be ridiculous. The fact that Taxi and Meta already have Combat Shotguns instead of riot shotguns is a mistake.
Ordering a ballistic crate or two at roundstart might be a good way to keep the armament up IC.

What about small caliber handguns?

**Comment by TermedClepe** [comment]
Armory is mostly irrelevant since wardens or HoS players very rarely if ever give out gun. In fact the only times armory is stripped clean to the point there's not enough guns in brig is when an antag raids it or it's a blob round.

What security needs to compete with antags and ghetto weapons is more access to departments to actually enforce space law regarding their manufacture.

**Comment by Imagebard** [comment]
>Also, what's the position of space law against non-sec having weapons?

Ghetto weapons(Spears and the shit) are considered minor sentences, actual weapons are big sentences. Using them usually means assault, murder, with blast cannon sabotage/grand sabotage for opening up the floor and destroying equipment.
Don't trust the "2ND AMENDMENT" shitters, pardoning crimes is above something security/the captain can do and is considered an Abuse of Power charge.

**Comment by viniciuspetch** [comment]
I agree with Termed, including the idea to give more access (secs can't stop a fistfight inside medbay because they can't enter it). Also, Termed's comment gave me two ideas:

- Give access to captain and another heads if there's no HoS nor warden, **or** in last instance give it to the entire sec. This way, we'll always have a way to open armory and distribute weapons. _But if the station doesn't return the weapons?_ Now comes the second idea:

- Give gps and remote control to security's weapons. This way, if someone steals or doesn't return the weapon, security can simply deactivate that weapon and trace it to the thief/nonreturner and retrieve it. Apart from being good or not, this second idea might be a pain in the ass to implement.

**Comment by PJB3005** [comment]
Also can we FUCKING REMOVE THE WINDOORS FROM BOX' ARMOURY?

Thanks.

But yeah sec needs more guns and it should be easier to hand them out.
Bay has something where the warden has a button that opens blast doors so the armoury is open to the entire security office, it'll need some redesigning of our brig to implement though (does anybody ever use the firing range anyways?)

**Comment by Imagebard** [comment]
>(does anybody ever use the firing range anyways?)

It has a advantage, I can see people trying to break in form space.
Not that I am often there to check, but its a upside for any Warden.

**Comment by viniciuspetch** [comment]
@Imagebard
The thing is, because of constant threat of antags:
-people always get a weapon for emergency.
-the station don't snitch weapon holders.
-sec never bothers with it because, well, only 4 of 20 weapon holders might actually use them for harmful intent, and enforcing it means the entire station under bars (or riot).
@PJB3005
Never saw anyone using the firing range, so goodbye (just leave a window there for the ocasional break-in). Just remember to relocate some items that spawn there, like a box of flashbangs. And we really should remove that windoors, aren't they access free?

**Comment by Imagebard** [comment]
You can use those windoors as long as you have basic security access, I believe.

**Comment by LordMyroc** [comment]
Honestly I think this is more of a problem with non-sec being too armed. People being able to make makeshift defense against antag is good, yes, but the weapons introduced in the Constructable Weapons PR are really pushing it. (Especially in the case of the Blast Cannon which might as well have ANTAG ONLY printed on it in large bold letters.)

Another thought, though, instead of buffing the armory, consider buffing the average sec officer. By default the weapons they get is a stun-only taser with a grand total capacity of 5 shots, as well as a stun baton that can easily be disarmed and used against them. In addition, their armor does nothing to protect them from the one-hit stuns that combat has centered around for years. It is a disturbingly frequent occurence to see scientists more well-armed than officers, which I think is a problem.

**Comment by Imagebard** [comment]
>In addition, their armor does nothing to protect them from the one-hit stuns that combat has centered around for years.

There's a "energy =" flag in a lot of armor resistances, and it decreases the time you stay stunned. Example form the engineering hardsuit:
(melee = 40, bullet = 5, laser = 20,**energy = 5**, bomb = 35, bio = 100, rad = 80)
At least, that's what I remember it being.
Maybe a alternate armor for casual security members, something that fits with the officer beret maybe.

**Comment by Shadowmech88** [comment]
Perhaps Security could get a machine that allows them to use raw materials to create customized, not-so-improvised guns and ammunition.

**Comment by viniciuspetch** [comment]
I think giving a "weapon protolathe", connected to Science's network is a better idea. The only thing left is removing Science's weapons without affecting the research.

**Comment by Imagebard** [comment]
Being able to craft Energy Pistols without having to ask Scientists I'll end up arresting later is a good idea.

**Comment by SonixApache** [comment]
Handguns to hand out to officers for code blue / red situations, ammolathe to produce munitions and simple weapons?

**Comment by 9600bauds** [comment]
I don't understand your grudge against the blast Cannon, it's a gimped bomb

**Comment by SangriaMonkey** [comment]
A rack of assault rifles would be nice. They really need more EMP-resistant gear.

**Comment by FudgePucker** [comment]
@9600bauds Maybe because having a gimped directional bomb is better than a normal undirectional bomb.

Oh hey AI is malf?
Don't have a good enough burn mix and can't get a space suit to set up more than one bomb?
Just make a blast cannon and shoot at the AI.

**Comment by Imagebard** [comment]
THERE GOES THE ISSUE.

I'm still suggesting having Ricochet Rifles there by default.

>A rack of of assault rifles

I'm not so sure about that, they're not Nanotrasen weapons. Maybe the SMGs R&D can build will work.
That, or we change the shotguns we have a little(something I would support): Sec should have 3 boxes of each ammo instead of 1 of each as we have right now (You can find those ammo boxes right before the entrance to the armory, over the Warden office). It should allow 3 officers to use the shotguns instead of just one.

**Comment by SangriaMonkey** [comment]
Shotguns are great but they're too bulky for every situation. It would be nice to have a compact alternative to them.

**Comment by Imagebard** [comment]
Oh wait, I thought you meant the Syndicate machineguns when I read "assault rifles".
I guess that's fine.

**Comment by SonixApache** [comment]
Actually yeah, R&D assault rifles would be a good idea.
Loading the riot shotguns pre-emptively and adding some more ammo might be a nice idea as well.

**Comment by NigglyWiggly** [comment]
@Imagebard 
>Don't trust the "2ND AMENDMENT" shitters

# SHALL NOT BE INFRINGED

<sub>There is no second amendment in Spess and you need a weapon permit on your ID to legally brandish a weapon.</sub>

**Comment by Zth--** [comment]
I have a great idea. Make it so making weapons is disabled in the protolathe at roundstart. And then make it so red code enables the weapons factory (or emag the protolathe)

**Comment by Shadowmech88** [comment]
But only two people with head access can trigger a code red. If the heads are dead or missing, even Security personnel won't have access to the protolathe weapons.

**Comment by SonixApache** [comment]
> There's actually no reason to produce guns if there's no threat.
Stocking the armory, upgrading sec gear, etc etc

**Comment by Zth--** [comment]
>Stocking the armory, upgrading sec gear, etc etc

I've been playing here for 2 years maybe? And I saw that happen maybe once? **MAYBE **once

>But only two people with head access can trigger a code red. If the heads are dead or missing, even Security personnel won't have access to the protolathe weapons.

I think that's actually a problem with code red itself. We can add a computer to the bridge that every head can use, which allows to enable maint access and/or weapon production.

**Comment by Kitfox88** [comment]
I like the idea of adding built in tracking and remote shutdown to weapons manufactured/starting on the station (with the ability to remove it or bypass it somehow with work). That way Security can track shitters who rob the armory and if R&D starts going full moran with their AEGs sec can lock em down and bustem.

Might be a pain in the ass to code though, idk.

**Comment by icantthinkofanameritenow** [comment]
Adding a copy or two of the weapons Deity added a while ago that don't spawn normally to the armory might work too, like the Combine Overwatch Rifle or the Nikita. 

Squaddie Armor is actually worse against everything but bullets compared to standard Sec armor, and the laser guns are all not much of an improvement other than the laser cannon. As for the weapon tracking idea, it's an ideal concept butnit would be hellishly hard to code and shitters will just use the constructable weapons instead. (And seeing that every other round half the station is walking around with swords, that's already happening now). The weapon-only protolathe exclusive to Sec will never work, either because the community will throw a shitfit over MUH GUNZ or because nobody will care enough in-game to supply it with materials and make the stuff needed to keep the armory stocked.

**Comment by Imagebard** [comment]
>Combine Overwatch Rifle

Maybe, but where the hell would they get the ammo.

>the Nikita

Why the fuck would Security have a Rocket Launcher?

**Comment by DrCelt** [comment]
This is an awful idea and anyone in favour of it is awful.

**Comment by Imagebard** [comment]
*Good thing we have a man of diplomacy and argumentation here, DrCelt will surely stop this idea dead on its tracks.*

**Comment by DrCelt** [comment]
Security have stun gun in their armoury. In the current stun system, which is awful, they are the most powerful weapon in the game second only to bullet weapons that stun, or perhaps shotguns if you can time it right. 
So, as for weapons in the armoury, they already have the most powerful non antagonist weapons (egun & shotgun) in the game. Why is there a need to give them more?

**Comment by SangriaMonkey** [comment]
They have precisely one weapon that doesn't get fucked over by EMPs in there.

**Comment by DrCelt** [comment]
That's not so much an issue with the weapons as it is with EMPs. They still have the most powerful non stunning weapon in the game even if we assume the enemy have unlimited EMPs.

**Comment by icantthinkofanameritenow** [comment]
@Imagebard

The Overwatch Rifle's standard ammo clips can be used in a recharger. Only the dark energy orbs are limited. 

And you know what? Why not add the Lawgiver in there too? Would be an appropriate addition for Security, or at least more so than the stormtrooper modkit.

**Comment by icantthinkofanameritenow** [comment]
@DrCelt

But there are no AEGs in the Armory.

**Comment by Imagebard** [comment]
>Why not add the Lawgiver in there too?

Are you out of your goddamn mind?

**Comment by icantthinkofanameritenow** [comment]
@ImageBard

Merely throwing out suggestions. If you have any other ideas, feel free to share them. 

At the very least, adding in at least one AEG would be wise.

**Comment by icantthinkofanameritenow** [comment]
Plus, simply adding things to the Armory won't help security in the long run, the crew's access to weapons has to be restricted too. I'm tempted to suggest that the constructable weapon update be rolled back as one way of making it happen, but as usual the server would never allow anything that keeps them from validhunting to be merged.

**Comment by Shadowmech88** [comment]
@icantthinkofanameritenow 
The lawgiver is a tad bit overpowered to be a stock armory spawn. Maybe a set could be requested from Centcomm during code red like an ERT, but I don't like the idea of it spawning in the armory by default.

**Comment by Probe1** [comment]
We've updated the armory and completely redone security on deff since the issue was discussed.
